ak_survey_results.py

```diff
@@ -511,13 +511,8 @@ def main(args):
                 'error': str(e)
             }
     elif args.FUNCTION == 'process_surveys_that_need_updating':
-        try:
-            surveys = ak.surveys_that_need_updating(15)
-            return ak.process_surveys_that_need_updating(surveys, args.LAMBDA)
-        except Exception as e:
```

Comment:
Should we do the same thing in other places in `main` where we have the same try/except pattern?

Reply:
Yes, made the change!